tests/prague/eip7620_eof_create/test_eofcreate.py

```diff
@@ -0,0 +1,470 @@
+"""
+Test good and bad EOFCREATE cases
+"""
+
+import pytest
+
+from ethereum_test_tools import (
+    Account,
+    Environment,
+    StateTestFiller,
+    TestAddress,
+    compute_eofcreate_address,
+)
+from ethereum_test_tools.eof.v1 import Container, Section
+from ethereum_test_tools.eof.v1.constants import NON_RETURNING_SECTION
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+from .helpers import (
+    default_address,
+    fixed_address,
+    simple_transaction,
+    smallest_initcode_subcontainer,
+    smallest_runtime_subcontainer,
+)
+from .spec import EOF_FORK_NAME
+
+REFERENCE_SPEC_GIT_PATH = "EIPS/eip-7620.md"
+REFERENCE_SPEC_VERSION = "52ddbcdddcf72dd72427c319f2beddeb468e1737"
+
+pytestmark = pytest.mark.valid_from(EOF_FORK_NAME)
+
+
+def test_simple_eofcreate(
+    state_test: StateTestFiller,
+):
+    """
+    Verifies a simple EOFCREATE case
+    """
+    env = Environment()
+    pre = {
+        TestAddress: Account(balance=10**21, nonce=1),
+        default_address: Account(
+            code=Container(
+                sections=[
+                    Section.Code(
+                        code=Op.SSTORE(0, Op.EOFCREATE[0](0, 0, 0, 0)) + Op.STOP,
+                        code_inputs=0,
+                        code_outputs=NON_RETURNING_SECTION,
+                        max_stack_height=4,
+                    ),
+                    Section.Container(container=smallest_initcode_subcontainer),
+                ],
+                data=b"abcdef",
+            )
+        ),
+    }
+    # Storage in 0 should have the address,
+    post = {
+        default_address: Account(
+            storage={
+                0: compute_eofcreate_address(default_address, 0, smallest_initcode_subcontainer)
```

Comment:
put the same storage in pre to some value to be protected from compute_eofcreate_address function returning empty